.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -85,20 +85,20 @@ jobs:
         go build -trimpath -o bin/updater ./cmd/updater
         go clean -cache -modcache
 
-        # TODO(ROX-23684): Ensure previous release versions can coexist with
-        #                  muti-bundle command line arguments.
-        ./bin/updater -output-dir=${{ env.ROX_PRODUCT_VERSION }}
+        mkdir ${{ env.ROX_PRODUCT_VERSION }}
 
-        # The vulnerability file created by older versions can remain the name
-        # vulns.json.zst in the GCS bucket.
-        #
-        # TODO(ROX-22285): Remove after 4.4
-        for file in output.json.{zst,ztd}; do
-          if [ -f "${{ env.ROX_PRODUCT_VERSION }}/${file}" ]; then
-            mv "${{ env.ROX_PRODUCT_VERSION }}/${file}" "${{ env.ROX_PRODUCT_VERSION }}/vulns.json.zst"
-            break
-          fi
-        done
+        # Run updater per release/product version.
+        case "${{ env.ROX_PRODUCT_VERSION }}" in
+        4.4.*)
+            ./bin/updater -output-dir="${{ env.ROX_PRODUCT_VERSION }}"
+            ;;
+        *)
+            ./bin/updater export --split bundles
+            zip ${{ env.ROX_PRODUCT_VERSION }}/vulnerabilities.zip bundles/*.json.zst
+            ;;
+        esac
```

Comment:
Thoughts on still publishing the single bundle in 4.5 as well as the multi/split bundle? Thinking if some unexpected 'bug' occurs with the split bundles, a user having the ability to toggle off the split bundles would buy some time to work out a fix.

Reply:
This is a tradeoff. Although your concern is valid, two bundle formats open the door for discrepancies and increase the workflow's complexity (e.g., more calls to the data sources, etc.). The good news is that we can always enable the single bundle on our end if needed without requiring any changes in the deployments (except, of course, users would have to flip the multi-bundle feature env to `false`).

Reply:
Ah yes, good point! Agreed we could react by enabling the single file bundle if needed.